presto-hive/src/main/java/io/prestosql/plugin/hive/HivePageSourceProvider.java

```diff
@@ -384,9 +385,12 @@ public int getIndex()
                     }
                 }
                 else {
+                    String partitionKeyValues = String.join("/", partitionKeys.stream()
+                            .map(partitionKey -> format("%s=%s", partitionKey.getName(), partitionKey.getValue()))
```

Comment:
Do we need some escaping of `/` or `=` here?

Reply:
Also wouln't `MAP` be a better type here instead of `VARCHAR`?

Reply:
I think escaping or encoding is needed. Let me update it. 

Exactly, `MAP` type is also a candidate for this column. 
@martint Do you have any opinion?

Reply:
there might be also be `:`-escaping and canonicalization of representation.

but, since we know partition name before hand, why dont we just pass it here, instead of reconsructing?
did you consider this approach?

cc @electrum

Reply:
Replaced with `partitionName` provided by Hive split.